cli/cmd/play.go

```diff
@@ -78,6 +79,29 @@ func internalPlayModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 			"or directory")
 	}
 
+	if !libconnect.IsBaseURI(args[0]) && !libconnect.IsCredentialsURI(args[0]) {
+		// FillCtx returns error if no instances found.
+		var runningCtx running.RunningCtx
+		if err := running.FillCtx(cliOpts, cmdCtx, &runningCtx, args, false); err != nil {
+			return util.InternalError(
+				"Internal error: could not parse application name: %s",
+				version.GetVersion, err)
+		}
+		if len(runningCtx.Instances) > 1 {
+			return util.InternalError(
+				"Internal error: specify instance name",
+				version.GetVersion)
+		}
+
+		_, err := os.Stat(runningCtx.Instances[0].BinaryPort)
+		if err != nil {
+			return util.InternalError(
+				"Internal error: application binary port does not exist: %s",
+				version.GetVersion, err)
+		}
+		args[0] = runningCtx.Instances[0].BinaryPort
+	}
```

Comment:
Please, do it in the connect manner. It will help us to have the same behavior for all modules:

https://github.com/tarantool/tt/blob/96db0688623bf0a8072b3c985eb20c3d1b5df617/cli/cmd/connect.go#L142-L187

```suggestion
	// FillCtx returns error if no instances found.
	var runningCtx running.RunningCtx
	var err error
	if err = running.FillCtx(cliOpts, cmdCtx, &runningCtx, args, false); err == nil {
		_, err := os.Stat(runningCtx.Instances[0].BinaryPort)
		if err != nil {
			return util.InternalError(
				"Internal error: application binary port does not exist: %s",
				version.GetVersion, err)
		}
		args[0] = runningCtx.Instances[0].BinaryPort
	} else if libconnect.IsCredentialsURI(args[0]) {
		// Fetch credentials from the URI.
	} else if libconnect.IsBaseURI(args[0]) {
		// Fetch credentials from ENV.
	} else {
		// Return wrapped err.
	}
```

Reply:
Updated.

Reply:
Updated.